src/shared/util.js

```diff
@@ -1219,16 +1219,111 @@ function MessageHandler(sourceName, targetName, comObj) {
   this.targetName = targetName;
   this.comObj = comObj;
   this.callbackIndex = 1;
+  this.streamId = 1;
```

Comment:
> but I now notice that this may become problematic with respect to the line length limit

"_line length limit_"? I think i am not getting this point, can you please elaborate?